include/LLGL/Container/DynamicArray.h

```diff
@@ -360,7 +360,7 @@ class LLGL_EXPORT DynamicArray
             return *this;
         }
 
-        DynamicArray& operator = (DynamicArray&& rhs)
+        DynamicArray& operator = (DynamicArray&& rhs) noexcept
```

Comment:
I think `DynamicArray` isn't a candidate for this either, because it can be used for any kind of elements. While it's not good practice for any class to throw exceptions in the destructor, we would have add a condition into `noexcept()` depending on whether its element type could potentially throw. The other classes look fine, but I think we should skip this one as well.